run-qa-checks

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 set -e
+
+jshint ./openwisp_notifications/static/openwisp_notifications/js/*.js
+
```

Comment:
can we also add linting for the CSS as we have in https://github.com/openwisp/OpenWISP-Website/blob/master/run-qa-checks#L6?